zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -121,6 +130,7 @@ object Request {
    * Constructor for Request
    */
   def apply(
+    version: Version = Version.`HTTP/1.1`,
```

Comment:
Make this as last parameter to minimize the diff

Reply:
Intentionally added as first parameter
So when we see any encoding for request of `Netty` , HttpVersion is always a first argument.
followed same order.

Reply:
Seems intuitive to me to have the version as first param.